hazelcast/src/main/java/com/hazelcast/jet/config/JetConfig.java

```diff
@@ -378,4 +378,20 @@ public int hashCode() {
         result = 31 * result + (resourceUploadEnabled ? 1 : 0);
         return result;
     }
+
+    @Override
+    public String toString() {
+        return "JetConfig{"
+                + "instanceConfig=" + instanceConfig
```

Comment:
there is no point in logging instanceConfig, it is deprecated and always DelegatingInstanceConfig